From 18bba65adf58d54645398de26eac24081d48f793 Mon Sep 17 00:00:00 2001
From: Angelo Compagnucci <angelo.compagnucci@gmail.com>
Date: Thu, 25 Feb 2016 22:19:31 +0100
Subject: [PATCH] mcs/class/monodoc/Makefile: Fixing wrong monodoc search path

mono_libdir actually leaks host compiler path, fixing
by hardcoding correct path for our usecase.

Signed-off-by: Angelo Compagnucci <angelo.compagnucci@gmail.com>
---
 mcs/class/monodoc/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mcs/class/monodoc/Makefile b/mcs/class/monodoc/Makefile
index 3ebba80..ddc2098 100644
--- a/mcs/class/monodoc/Makefile
+++ b/mcs/class/monodoc/Makefile
@@ -135,7 +135,7 @@ test-local: setup-doc-sources
 dist-local: Monodoc.Ecma/EcmaUrlParser.cs
 
 $(the_lib).config: Makefile monodoc.dll.config.in
-	sed 's,@monodoc_refdir@,$(mono_libdir)/monodoc,g' monodoc.dll.config.in > $@
+	sed 's,@monodoc_refdir@,/usr/lib/monodoc,g' monodoc.dll.config.in > $@
 
 Monodoc.Ecma/EcmaUrlParser.cs: Monodoc.Ecma/EcmaUrlParser.jay $(topdir)/jay/skeleton.cs jay.sh
 	$(topdir)/$(thisdir)/jay.sh $(topdir) $< $@ $(JAY_FLAGS)
-- 
1.9.1

